langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -119,6 +157,12 @@ public isolated function getName(Element elem) returns string = @java:Method {
 
 # Changes the name of an XML element.
 #
+# ```ballerina
+# xml:Element s = xml `<person>John</person>`;
+# s.setName("student");
+# s ⇒ <student>John</student>
```

Comment:
Shall we write this in a single line and also add an invalid case? Please do this wherever applicable in all other examples. Refer [[1]](https://github.com/ballerina-platform/ballerina-lang/pull/39130/files#r1060471733)

Reply:
The `.setName()` would not return a value. Therefore, a variable is used to display the output.